test/ae_mdw/sync/block_test.exs

```diff
@@ -0,0 +1,81 @@
+defmodule AeMdw.Sync.BlockTest do
```

Comment:
Shouldn't this map the module that it is testing? `AeMdw.Db.Sync.Block` => `AeMdw.Db.Sync.BlockTest`